src/main/java/edu/harvard/iq/dataverse/dataset/DatasetUtil.java

```diff
@@ -109,7 +109,7 @@ public static DatasetThumbnail getThumbnail(Dataset dataset, DatasetVersion data
             if (dataAccess.getAuxFileAsInputStream(datasetLogoThumbnail + thumb48addedByImageThumbConverter) != null) {
                 in = dataAccess.getAuxFileAsInputStream(datasetLogoThumbnail + thumb48addedByImageThumbConverter);
             }
-        } catch (Exception ioex) {
+        } catch (IOException ioex) {
```

Comment:
IOExceptions are just for input / output and can't handle null pointers, which this code does. While most changes in this commit are definitely solid we'll have to revert this one I think -- as Travis CI build is failing (confirmed via clean & build on my machine).